### DISCUSSION TITLE
ready for usage or hold off?

### DISCUSSION THREAD

**Discussion by sheldonhull** [pomdtr/sunbeam#33] [Q&A]
@pomdtr In the Charm community a while back, you had said to hold off as still making significant changes.
Is it ready for jumping into using this more now?
Love what I see so far. Thanks for this project 👋🏻

**Comment by pomdtr** [comment]
Yeah the api it is quite mature now, you can take it for a spin. If I do some changes to the output schema they will be quite limited, and I'll communicate them on the changelog.

I'm mostly working on dev exp right now, especially for simple extensions.

Feel free to contribute if you find any bugs, and give your feedback.

Thanks for the kind words!
